src/main/java/frc/robot/subsystems/Pitch.java

```diff
@@ -56,6 +64,7 @@ public void configure() {
     pitchConfig.MotorOutput.NeutralMode = constPitch.PITCH_NEUTRAL_MODE_VALUE;
 
     pitchMotor.getConfigurator().apply(pitchConfig);
+    pitchMotor.getConfigurator().apply(pitchCurrentLimitConfig);
```

Comment:
Instead of making a separate configurator, just add these configs to the pitchConfig
Ex.
```java
    pitchConfig.CurrentLimits.SupplyCurrentLimitEnable = prefPitch.enablePitchSupplyCurrentLimit.getValue();

```

Reply:
nuh uh